lasagne/layers/dnn.py

```diff
@@ -24,7 +24,49 @@ class DNNLayer(Layer):
 
 
 class Pool2DDNNLayer(DNNLayer):
+    """
+    2D pooling layer
 
+    Performs 2D mean- or max-pooling over the two trailing axes of a 4D input
+    tensor. This is an alternative implementation which uses
+    ``theano.sandbox.cuda.dnn.dnn_pool`` directly.
+
+    Parameters
+    ----------
+    incoming : a :class:`Layer` instance or tuple
+        The layer feeding into this layer, or the expected input shape. The
+        output of this layer should be a 4D tensor.
+
+    pool_size : integer or iterable
+        The length of the pooling region in each dimension
```

Comment:
I guess it could also be a list or another iterable with 2 elements? :) I guess this also needs clarification in the original pooling layer docs then.